build/scripts/Automations/HowToAddAnAutomation.md

```diff
@@ -0,0 +1,48 @@
+# How to add a new automation
+
+1. Create a folder in `build\scripts\Automations`. The name of the folder is the name of the automation, e.g. "_UpdateAppBaselines_".
+1. Create a `run.ps1` file in the new folder. It should contain the script with the automation's logic. The script can accept the following parameters:
```

Comment:
Missing some doc on what a run.ps1 should return (if anything)

Reply:
Added doc